test/elevation.c

```diff
@@ -0,0 +1,23 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Fair enough, I'll change it. Since different source files have older/newer style boilerplate it might be an idea to update [`CONTRIBUTING.md`](https://github.com/libuv/libuv/blob/v1.x/CONTRIBUTING.md) to point to the correct one as currently all it says is
```
Look at other tests to see how they should be structured (license boilerplate, the way entry points are declared, etc.).
```